### PR TITLE
docs: change old contact link to the Slack one

### PR DESCRIPTION
**Pull request by jorgehermo9**
Inside of the Matrix channel, it is mentioned that the official contact channel is now Slack.

### CODE REVIEW

**Comment by mervyn-mccreight** [comment]
Thanks for your contribution 🚀 .
I think it would be a nice idea to also give a hint about the channel inside that slack instance (testcontainers-rust), WDYT?

**Comment by jorgehermo9** [comment]
done! Changed a little bit the link handling with the markdown link embedding, what do you think?

**Review by mervyn-mccreight — APPROVED** [review]

**Comment by jorgehermo9** [comment]
Waiting for someone with write access to merge :)

**Comment by mervyn-mccreight** [comment]
Yeah will do that today :)

**Comment by mervyn-mccreight** [comment]
I noticed there is a badge linking to the matrix chat in the readme as well, would you mind to replace this as well with a nice slack badge linking to the testcontainers slack?

**Comment by jorgehermo9** [comment]
missed that. Yes! I'll look into it 😀

**Review by DDtKey — APPROVED** [review]

**Comment by DDtKey** [comment]
LGTM once `readme.md` is updated as well, thank you!

**Comment by jorgehermo9** [comment]
Thanks for your review, looking forward to that as well. Couldn't find time these days, I'll commit the changes asap😁

**Comment by jorgehermo9** [comment]
Done, the badge is added, do you like it or should I change something? The badge links to an invitation to the slack workspace. 

Also, couldn't find the remaining matrix mention in the `CONTRIBUTING.md`, I think I replaced the only one already in this PR, check [this file](https://github.com/jorgehermo9/testcontainers-rs/blob/broken-contact-link/CONTRIBUTING.md) (maybe I'm mistaken)

**Review by DDtKey — APPROVED** [review]

**Comment by DDtKey** [comment]
Looks good, looks like all matrix mentions have been updated
Thank you!

**Pull request merged**
